tests/unit_tests/simpll/SimpLLTest.cpp

```diff
@@ -1,6 +1,35 @@
+#include "Logger.h"
+
+#include <cerrno>
 #include <gtest/gtest.h>
+#include <ostream>
+
+/// Sets logging verbosity based on `SIMPLL_VERBOSITY` env. var.
+/// Returns non-zero if the value is invalid.
+int setVerbosity() {
```

Comment:
In some languages (especially C#) it is common that functions that return pass/fail status and do some work in their body to have `try` prefix in front of their names. I suggest renaming this to `trySetVerbosity` and to return boolean.

Then in main we could return `EXIT_SUCCESS` or `EINVAL` directly.